tests/components/vesync/snapshots/test_switch.ambr

```diff
@@ -237,6 +267,36 @@
 # ---
 # name: test_switch_state[Humidifier 600S][devices]
   list([
+    DeviceRegistryEntrySnapshot({
```

Comment:
Why does adding a new platform create all these snapshots?

Reply:
How are humidifers now added?

Reply:
They aren't.  Humidifier devices are skipped and not displayed.   A few github issues exist for this.  This PR represents the first platforms being used for sensors. Another PR exists to add the humidifier platform as well.